### PR TITLE
#12 test survey structure

### PR DESCRIPTION
**Pull request by fudw**
Created test survey on LimeSurvey with all question types for xml parser testing purpose. Added two files containing the test survey structure. Updated data_description.md.

### CODE REVIEW

File: data/data_description.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 - survey_structure.xml: Question and answer structures of the survey are described connecting the codes with the text
 - dummy_data_codeonly.csv: Dummy data from 2020 using only the question codes for the column headers
 - dummy_data_codeandtext.csv: Dummy data from 2020 using the code and the question text for the column headers
+- test-survey-structure.ssl: Test survey with all question types; for LimeSurvey import
```
Comment (olineumann) [inline data/data_description.md:4]: Isn't it *.lss?

File: data/test_survey_structure.xml [hunk 2]
```diff
@@ -0,0 +1,832 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (olineumann) [inline data/test_survey_structure.xml:1]: This XML Structure is given by LimeSurvey by exporting surveys right?

**Review by olineumann — COMMENTED** [review]
If the `test-survey-structure.xml` is generated automatically by lime survey and the `test-survey-structure.ssl` is the survey it looks good to me. Maybe we have to note in the `data_description.md` what version of LimeSurvey we used because maybe they change the XML structure or importing the `test-survey-structure.ssl` will fail in newer versions.

File: data/data_description.md (hunk 1 above)
Comment (fudw) [reply data/data_description.md]: Oops. Corrected the typo.

File: data/test_survey_structure.xml (hunk 2 above)
Comment (fudw) [reply data/test_survey_structure.xml:1]: Genau.

**Comment by fudw** [comment]
> Maybe we have to note in the `data_description.md` what version of LimeSurvey we used because maybe they change the XML structure or importing the `test-survey-structure.ssl` will fail in newer versions.

Added version number in "data_description.md"

**Review by olineumann — APPROVED** [review]

**Pull request merged**